cdap-examples/LogAnalysis/bin/inject-data.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+
+#
+# Copyright © 2015 Cask Data, Inc.
```

Comment:
i thought we wanted to move away from these scripts and just use cdap-cli for ingestion.